help/develop/how-to/report-tickets.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Report bugs and feature requests"
+---
+
+This guide provides an easy set of rules for reporting issues and managing
```

Comment:
Get rid of "easy", it's unnecessary and doesn't sound well, too (the author probably meant "simple") :)